### ISSUE TITLE
A discussion topic?

### ISSUE THREAD

**Issue by afomi** [civichackingagency/request-support#1]
Could this live in [Discussions](https://github.com/civichackingagency/discussions)?

I'm thinking about:

1. one less repo to manage, maintain, and navigate to
2. the user of experience of raising a hand in a Discussion thread vs the affordances of a repo, which might lead to forks and duplicates; which I'm not clear on the desirability/value of

**Comment by lukefretwell** [comment]
@afomi thoughts:

* Would it be helpful to have people 'spec' in some way? I was thinking that creating an issue template for this to help folks understand the need (and help the requestor have some way to convey their needs).
* I figured using a repo would let people follow to see what new requests come in.

Workflow:

1. Submit an request support issue.
2. CHA promotes or folks following get notified.
3. Discussion ensues, folks coordinate, repo (or repos) is created.
4. Issue is closed.

Now that I've written this out, I'm going to use the '[Pitches](https://github.com/orgs/civichackingagency/discussions/categories/pitches)' discussion topic I created and we can figure out how to direct folks to engage their and how to move forward (aka project template).
